content/docs/guides/10-minute-tutorial.md

```diff
@@ -177,6 +177,8 @@ public class RunCucumberTest
 }
 ```
 
+Create an empty directory `src/test/resources/hellocucumber` which later will contain the feature files.
```

Comment:
Afaik this directory should be created by the archetype?

Reply:
Just wanted to mention, I opened a PR for this piece already: https://github.com/cucumber/docs.cucumber.io/pull/425